quinn-proto/src/connection/mod.rs

```diff
@@ -1865,8 +1865,11 @@ impl Connection {
             None
         };
 
+        let stateless_reset = is_stateless_reset(partial_decode.data(), &self.peer_params);
```

Comment:
This introduces a single-caller function. In this case, why is that better than introducing a block of code inline here, maybe with some comments to demarcate different parts?

Reply:
At time of writing I thought it might end up with two callsites. Inlined.